app/controllers/api/v1/lois_controller.rb

```diff
@@ -65,7 +69,28 @@ def create
       end
 
       if @loi.submitted
-        Mail.new( :to => @loi.email, :from => 'colleen@svpchicago.org', :subject => 'Submission to SVP received', :body => File.read('app/views/submission_email.html.erb')).deliver!
+        Mail.new( 
+          :to => @loi.email, 
+          :from => 'colleen@svpchicago.org', 
+          :subject => 'Submission to SVP received', 
+          :body => File.read('app/views/submission_email.html.erb'),
+          :content_type => 'text/html; charset=UTF-8'
+        ).deliver!
+
+        # The link that gets generated will need to be changed when deploying to production
+        File.open('app/views/loi_submitted_email_to_admin.html.erb', 'w') { |file| file.write(
+        "<p>A user has submitted an LOI form</p>
+        <p>Click below to see it</p>
+        <p><a href='https://demo-svp-chicago.herokuapp.com/lois/#{@loi.id}'>View LOI</a></p>"
```

Comment:
Sweet!

On Thu, Aug 18, 2016 at 10:47 PM, Dan Hey <notifications@github.com> wrote:

> In app/controllers/api/v1/lois_controller.rb
> <https://github.com/difference-engine-svp/svp-chicago/pull/129#discussion_r75424297>
> :
>
> > @@ -65,7 +69,28 @@ def create
> >        end
> >
> >        if @loi.submitted
> > -        Mail.new( :to => @loi.email, :from => 'colleen@svpchicago.org', :subject => 'Submission to SVP received', :body => File.read('app/views/submission_email.html.erb')).deliver!
> > +        Mail.new(
> > +          :to => @loi.email,
> > +          :from => 'colleen@svpchicago.org',
> > +          :subject => 'Submission to SVP received',
> > +          :body => File.read('app/views/submission_email.html.erb'),
> > +          :content_type => 'text/html; charset=UTF-8'
> > +        ).deliver!
> > +
> > +        # The link that gets generated will need to be changed when deploying to production
> > +        File.open('app/views/loi_submitted_email_to_admin.html.erb', 'w') { |file| file.write(
> > +        "<p>A user has submitted an LOI form</p>
> > +        <p>Click below to see it</p>
> > +        <p><a href='https://demo-svp-chicago.herokuapp.com/lois/#{@loi.id}'>View LOI</a></p>"
>
> Great work.
> I think here, there's a way to dynamically set the base url so it can be
> something like BASE_URL + "/lois/#{@loi.id}
>
> I'm not sure how, but I'll look into it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/difference-engine-svp/svp-chicago/pull/129/files/bf12e88145b58bd33b3d5cb2a218372cab56c38b#r75424297>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQdZhlLY6DX6v4Ew75vopuYKbgVbcq_Dks5qhSdFgaJpZM4JoEUX>
> .
>